sproul-club/sc-frontend
# SCF 43: kanban to component

[SCF-43:](https://scfrontend.atlassian.net/browse/SCF-43?atlOrigin=eyJpIjoiNDk4NDgyMTQ4YzBjNDI1NWFlNmY0N2YyNjQ1MGEyMDgiLCJwIjoiaiJ9)

Steps to test: 
1. Drag the kanban things around and observe them still working :)

Screenshots: 
It literally looks the same please don't make me screenshot this
